packages/components/src/modal/stories/index.story.tsx

```diff
@@ -91,6 +87,7 @@ const Template: StoryFn< typeof Modal > = ( { onRequestClose, ...args } ) => {
 export const Default: StoryFn< typeof Modal > = Template.bind( {} );
 Default.args = {
 	title: 'Title',
+	style: { maxWidth: '600px' },
```

Comment:
As noted in [this comment](https://github.com/WordPress/gutenberg/pull/54471#issuecomment-1725088405), while this `max-width` rule makes the modal look more proportioned, it interferes with the `isFullScreen` prop (and with the new `contentWidth` prop). Probably best to remove it.